pkg/octtrpc/interceptor.go

```diff
@@ -100,6 +100,11 @@ func ServerInterceptor(opts ...Option) ttrpc.UnaryServerInterceptor {
 	}
 	return func(ctx context.Context, unmarshal ttrpc.Unmarshaler, info *ttrpc.UnaryServerInfo, method ttrpc.Method) (_ interface{}, err error) {
 		name := convertMethodName(info.FullMethod)
+		// Do not create spans for stats calls as they could be called
```

Comment:
This is hacky but not sure there is any other way if we want to necessarily drop only the stat spans coming from this interceptor. We could avoid this and possibly atleast drop the repetitive span coming from the service.Stats() call.

Reply:
I think this Interceptor is also creating duplicate spans since most of the APIs in `service.go` already create their own spans? Maybe we should just keep the spans in service.go and remove this interceptor? That way controlling what spans we want will be easy, plus those spans tend to include additional information (like container ID etc.)

Reply:
Yeah I did think of this and found this conversation in the original PR that added it https://github.com/microsoft/hcsshim/pull/678#issuecomment-524600648 so left it as is. Can this be revisited @kevpar ?

Reply:
Need to add a test case for stats to the interceptor tests added in #2330